File: DuDoong-Api/src/main/java/band/gosrock/api/comment/model/request/CreateCommentRequest.java
package band.gosrock.api.comment.model.request;


import javax.validation.constraints.NotBlank;
import javax.validation.constraints.Size;
import lombok.Getter;

@Getter
public class CreateCommentRequest {

    @NotBlank(message = "작성자 닉네임을 입력해주세요.")
    @Size(min = 1, max = 15)
    private String nickName;

    @NotBlank(message = "댓글 내용을 입력해주세요.")
    @Size(min = 1, max = 200)
    private String content;
}

File: DuDoong-Api/src/main/kotlin/band/gosrock/api/comment/model/request/CreateCommentRequest.kt
package band.gosrock.api.comment.model.request

import javax.validation.constraints.NotBlank
import javax.validation.constraints.Size

data class CreateCommentRequest(
    @field:NotBlank(message = "작성자 닉네임을 입력해주세요.")
    @field:Size(min = 1, max = 10)
    val nickName: String,

    @field:NotBlank(message = "댓글 내용을 입력해주세요.")
    @field:Size(min = 1, max = 150)
    val content: String,
)
